ocaml/libs/http-lib/http_svr.ml

```diff
@@ -713,78 +713,6 @@ let read_body ?limit req bio =
       else
         Buf_io.really_input_buf ~timeout:Buf_io.infinite_timeout bio length
 
-module Chunked = struct
```

Comment:
Oof, this is used in vhd-tool. with its won copy of the code, of course.

Reply:
Ah is it a copy of http_svr... I did see the name, but thought it was something else.

Reply:
The implementation of `ocaml/vhd-tool/src/chunked.ml` looks different. Crucially, it doesn't use `Buf_io`.